etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/TransformationRuleEngine.java

```diff
@@ -53,6 +54,7 @@ public void ensureRulesLoaded() throws RuleLoaderException {
                             ruleLoader.loadRules(resourceStream, new TypeReference<>() {});
                     rules.addAll(parsedRules);
                     rulesLoaded = true;
+                    IOUtils.closeQuietly(resourceStream);
```

Comment:
I wouldn't use this.  Use the "try with resources" mechanism.  I.e. the same thing that @basiliskus mentioned [below](https://github.com/CDCgov/trusted-intermediary/pull/1234/files#r1711680750).

Reply:
For the `catch`, I would catch `IOException | NullPointerException` and throw a `throw new RuleLoaderException("File not found: " + ruleDefinitionsFileName, new FileNotFoundException());` like above.